product_docs/docs/biganimal/release/overview/05_database_version_policy.mdx

```diff
@@ -7,19 +7,19 @@ title: "Database version policy"
 
 | **Postgres type**  | **Major versions**  | 
 | ----- | ------------------------- |
-| PostgreSQL  | 11 - 13  |
-| EDB Postgres Advanced Server  | 11 - 13  |
+| PostgreSQL  | 11–13  |
+| EDB Postgres Advanced Server  | 11–13  |
 
 
 ## Major version support
 
-PostgreSQL and EDB Postgres Advanced Server major versions are supported from the date they are made available until the version is retired by EDB (generally 5 years). See [Platform Compatibility ](https://www.enterprisedb.com/platform-compatibility#epas) for more details.
+PostgreSQL and EDB Postgres Advanced Server major versions are supported from the date they are made available until the version is retired by EDB (generally five years). See [Platform Compatibility ](https://www.enterprisedb.com/platform-compatibility#epas) for more details.
 
 
 
 ## Minor version support
 
-EDB performs periodic maintenance to ensure stability and security. EDB automatically performs minor version upgrades and patch updates as part of periodic maintenance. Customers are notified within the BigAnimal portal prior to maintenance occurring. Minor versions are not user configurable. 
+EDB performs periodic maintenance to ensure stability and security. EDB performs minor version upgrades and patch updates as part of periodic maintenance. Customers are notified in the BigAnimal portal prior to maintenance occurring. You cannot configure minor versions. 
```

Comment:
I know we frown on adverbs, but I think "automatically" provides some value in this paragraph.